app/lib/features/onboarding/pages/intro_page.dart

```diff
@@ -82,11 +82,11 @@ class IntroPage extends StatelessWidget {
           children: [
             RichText(
               text: TextSpan(
-                text: L10n.of(context).introPageDescription('desc1'),
+                text: L10n.of(context).introPageDescription1,
```

Comment:
this desc1, desc2, desc3 doesn't really work well for me ... I am wondering if we instead want to offer some  more semantic info or allow for a more comprehensive form of allowing to write entire paragraphs ... but it's fine for now